### DISCUSSION TITLE
Feature Request: Supporting collapsing multi-line log event

### DISCUSSION THREAD

**Discussion by ethernoy** [grafana/grafana#59004] [Observability]
Log entries may contain multiple lines from time to time. It would be very inconvenient to browse a stream of such log events if all events are expanded. I suggest we add a **collapse log event feature** in log explore page that helps to collapse this kind of event to facilitate log browsing.
![image](https://user-images.githubusercontent.com/15870761/202949119-704fdf2f-19c2-4f26-be40-89064f2e70b3.png)

Related issue: https://github.com/grafana/grafana/issues/58556

I am glad to help if somebody can show me where to modify :)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
